### PR TITLE
Add final modifier to classes marked as @final

### PR DESCRIPTION
**Pull request by franmomu**
## Subject

I am targeting this branch, because this breaks BC.

## Changelog

```markdown
### Changed
- Add final modifier to classes marked as `@final`.
```

### CODE REVIEW

File: src/Datagrid/DatagridMapper.php [hunk 1]
```diff
@@ -22,21 +22,18 @@
...
-    /**
-     * @var DatagridInterface
-     */
-    protected $datagrid;
-
     /**
      * @var DatagridBuilderInterface
      */
     protected $builder;
+    /**
```
Comment (OskarStark) [inline src/Datagrid/DatagridMapper.php:33]: Please add a blank line

File: src/Form/FormMapper.php [hunk 2]
```diff
@@ -23,21 +23,18 @@
...
-    /**
-     * @var FormBuilderInterface
-     */
-    protected $formBuilder;
-
     /**
      * @var FormContractorInterface
      */
     protected $builder;
+    /**
```
Comment (OskarStark) [inline src/Form/FormMapper.php:34]: Please add a blank line

File: src/Show/ShowMapper.php [hunk 3]
```diff
@@ -22,21 +22,18 @@
...
-    /**
-     * @var FieldDescriptionCollection
-     */
-    protected $list;
-
     /**
      * @var ShowBuilderInterface
      */
     protected $builder;
+    /**
```
Comment (OskarStark) [inline src/Show/ShowMapper.php:33]: Please add blank line

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Comment by franmomu** [comment]
Shall we add https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.18/doc/rules/class_notation/class_attributes_separation.rst adding `property`?

**Review by OskarStark — APPROVED** [review]

**Pull request merged**